src/styles/container.scss

```diff
@@ -0,0 +1,26 @@
+.container {
+  padding: 50px 40px;
+  margin: auto;
+  display: grid;
+  width: 944px;
+  height: 862px;
```

Comment:
Setting a fixed width and height for the container might cause layout issues on smaller screens. Consider using relative units or percentages to make the layout more flexible.